server/src/main/java/org/elasticsearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -478,7 +478,7 @@ private ClusterState applyCreateIndexRequestWithV2Template(final ClusterState cu
                                                                final String templateName,
                                                                final BiConsumer<Metadata.Builder, IndexMetadata> metadataTransformer)
                                                                                     throws Exception {
-        logger.info("applying create index request using v2 template [{}]", templateName);
+        logger.debug("applying create index request using the composable template [{}]", templateName);
```

Comment:
I think we can drop the "the"

```suggestion
        logger.debug("applying create index request using composable template [{}]", templateName);
```